src/frameName.cpp

```diff
@@ -177,6 +194,14 @@ void FrameName::javaMethodName(jmethodID method) {
         (err = jvmti->GetClassSignature(method_class, &class_name, NULL)) == 0) {
         // Trim 'L' and ';' off the class descriptor like 'Ljava/lang/Object;'
         javaClassName(class_name + 1, strlen(class_name) - 2, _style);
+        if (_includemm) {
+            jvmti->GetMethodModifiers(method, &modifiers);
+            for (std::map<int, std::string>::const_iterator iterator = access_flags.begin(); iterator != access_flags.end(); iterator++) {
+                if (modifiers & iterator->first) {
+                    _str.append(".").append(iterator->second);
```

Comment:
UX wise - this will create method names like `Fibonacci.private.static.fibonacci(I)J_[j]`. The [ticket](https://github.com/Granulate/gprofiler/issues/570) suggests we maintain such a format: `public final String.replace(....);` so in this case it should have been `private static Fibonacci.fibonacci(I)J_[j]`, can change it that way?